R/002_plot_components.R

```diff
@@ -60,7 +60,7 @@ create_rawplot_components <- function(proportional,
   return(plot_component)
 }
 
-#' Generates list of TRUE/FALSE for delta plot components that will be built
```

Comment:
I actually think this built way of phrasing is good. Think the one to change is the ones below for the function part.